Fix TestHiveGlueMetadataListing.createQueryRunner failure

https://github.com/trinodb/trino/actions/runs/10855433827/job/30128033072

```
Error:  io.trino.plugin.hive.TestHiveGlueMetadataListing -- Time elapsed: 11.41 s <<< ERROR!
io.trino.testing.QueryFailedException: Invalid location URI: file:///tmp/TrinoTest8209102461968800437/hive_data/
	at io.trino.testing.AbstractTestingTrinoClient.execute(AbstractTestingTrinoClient.java:134)
	at io.trino.testing.DistributedQueryRunner.executeInternal(DistributedQueryRunner.java:565)
	at io.trino.testing.DistributedQueryRunner.execute(DistributedQueryRunner.java:548)
	at io.trino.testing.QueryRunner.execute(QueryRunner.java:82)
	at io.trino.plugin.hive.TestHiveGlueMetadataListing.createQueryRunner(TestHiveGlueMetadataListing.java:74)
	at io.trino.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:113)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:507)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1489)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:2071)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:2033)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:187)
	Suppressed: java.lang.Exception: SQL: CREATE SCHEMA test_tpch_schema_wmb3rvbxxn WITH (location = 'file:///tmp/TrinoTest8209102461968800437/hive_data/')
		at io.trino.testing.DistributedQueryRunner.executeInternal(DistributedQueryRunner.java:572)
		... 11 more
Caused by: io.trino.spi.TrinoException: Invalid location URI: file:///tmp/TrinoTest8209102461968800437/hive_data/
	at io.trino.plugin.hive.HiveMetadata.lambda$createSchema$22(HiveMetadata.java:968)
	at java.base/java.util.Optional.map(Optional.java:260)
	at io.trino.plugin.hive.HiveMetadata.createSchema(HiveMetadata.java:963)
	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.createSchema(ClassLoaderSafeConnectorMetadata.java:410)
	at io.trino.tracing.TracingConnectorMetadata.createSchema(TracingConnectorMetadata.java:340)
	at io.trino.metadata.MetadataManager.createSchema(MetadataManager.java:787)
	at io.trino.tracing.TracingMetadata.createSchema(TracingMetadata.java:374)
	at io.trino.execution.CreateSchemaTask.internalExecute(CreateSchemaTask.java:128)
	at io.trino.execution.CreateSchemaTask.execute(CreateSchemaTask.java:82)
	at io.trino.execution.CreateSchemaTask.execute(CreateSchemaTask.java:54)
	at io.trino.execution.DataDefinitionExecution.start(DataDefinitionExecution.java:146)
	at io.trino.execution.SqlQueryManager.createQuery(SqlQueryManager.java:272)
	at io.trino.dispatcher.LocalDispatchQuery.startExecution(LocalDispatchQuery.java:150)
	at io.trino.dispatcher.LocalDispatchQuery.lambda$waitForMinimumWorkers$2(LocalDispatchQuery.java:134)
	at io.airlift.concurrent.MoreFutures.lambda$addSuccessCallback$12(MoreFutures.java:570)
	at io.airlift.concurrent.MoreFutures$3.onSuccess(MoreFutures.java:545)
	at com.google.common.util.concurrent.Futures$CallbackListener.run(Futures.java:1137)
	at io.trino.$gen.Trino_testversion____20[240](https://github.com/trinodb/trino/actions/runs/10855433827/job/30128033072#step:5:241)913_202817_284.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1570)
Caused by: java.lang.IllegalArgumentException: No factory for location: file:///tmp/TrinoTest8209102461968800437/hive_data/
	at io.trino.filesystem.manager.FileSystemModule.lambda$createFileSystemFactory$2(FileSystemModule.java:140)
	at java.base/java.util.Optional.orElseThrow(Optional.java:403)
	at io.trino.filesystem.manager.FileSystemModule.lambda$createFileSystemFactory$3(FileSystemModule.java:140)
	at io.trino.filesystem.switching.SwitchingFileSystem.fileSystem(SwitchingFileSystem.java:162)
	at io.trino.filesystem.switching.SwitchingFileSystem.directoryExists(SwitchingFileSystem.java:122)
	at io.trino.filesystem.tracing.TracingFileSystem.lambda$directoryExists$5(TracingFileSystem.java:128)
	at io.trino.filesystem.tracing.Tracing.withTracing(Tracing.java:47)
	at io.trino.filesystem.tracing.TracingFileSystem.directoryExists(TracingFileSystem.java:128)
	at io.trino.plugin.hive.HiveMetadata.lambda$createSchema$22(HiveMetadata.java:965)
	... 20 more
```

--- a/plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveGlueMetadataListing.java
+++ b/plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveGlueMetadataListing.java
@@ -14,6 +14,7 @@
 package io.trino.plugin.hive;
 
 import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
 import io.airlift.log.Logger;
 import io.opentelemetry.api.OpenTelemetry;
 import io.trino.Session;
@@ -69,7 +70,7 @@ public class TestHiveGlueMetadataListing
 
         this.glueMetastore = createTestingGlueHiveMetastore(dataDirectory, this::closeAfterClass);
         queryRunner.installPlugin(new TestingHivePlugin(dataDirectory, glueMetastore));
-        queryRunner.createCatalog(HIVE_CATALOG, "hive");
+        queryRunner.createCatalog(HIVE_CATALOG, "hive", ImmutableMap.of("fs.hadoop.enabled", "true"));
 
         queryRunner.execute("CREATE SCHEMA " + tpchSchema + " WITH (location = '" + dataDirectory.toUri() + "')");
         copyTpchTables(queryRunner, "tpch", TINY_SCHEMA_NAME, hiveSession, ImmutableList.of(TpchTable.REGION, TpchTable.NATION));

